### ISSUE TITLE
macOS preview and FinalCut Pro import

### ISSUE THREAD

**Issue by nogod4me** [avsaase/walksnail-osd-tool#45]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Software version

dev (2562d55)

### What operating system are you using?

macOS Sonoma 14.3.1

### What GPU are you using?

Apple Silicon M1

### What problem did you encounter?

Files, exported from the tool does not support preview mode in macOS (with space bar key), also there is no thumbnail and it does not imports to the FinalCut Pro. It plays but only with regular video player such as VLC, Quicktime.

**Comment by avsaase** [comment]
Yes I am aware. I also ran into the problem of preview mode not working. I was unable to find the cause of the problem. I don't have access to a Mac anymore so it's unlikely that I will be able to fix this. Hopefully someone else will come by and take a look at this.
